.carve/ignore

```diff
@@ -40,3 +40,4 @@ athens.common-events.graph.schema/valid-atomic-op?
 athens.common-events.graph.schema/explain-atomic-op
 athens.style/unzoom
 athens.self-hosted.fluree.test-helpers/query
+athens.common-db/get-block-property-document
```

Comment:
This fn is unused right now, but I expect it will be used when prototypes start using properties.

Reply:
Tested it, not unused anymore.